site/docs/guide.md

```diff
@@ -1082,9 +1082,38 @@ The first step in analyzing the performance of your build is to profile your
 build with the [`--profile`](user-manual.html#flag--profile) flag.
 
 The file generated by the [`--profile`](user-manual.html#flag--profile) flag is
-a binary file. Once you have generated this binary profile, you can analyze it
-using Bazel's [`analyze-profile`](user-manual.html#analyze-profile') command. By
-default, it will print out summary analysis information for each of the
+a JSON file. Once you have generated this JSON profile, you can analyze it
+using Chrome's [`chrome://tracing`](https://dev.chromium.org/developers/how-tos/trace-event-profiling-tool) tool or Bazel's
+[`analyze-profile`](user-manual.html#analyze-profile') command.
+
+### Chrome's `chrome://tracing`
```

Comment:
This is described here: https://docs.bazel.build/versions/master/skylark/performance.html#performance-profiling

We need to work on consolidating the docs on performance (see also https://github.com/bazelbuild/bazel/issues/11384)

Reply:
Oohhh, thanks! Didn't search hard enough :sweat: At a minimum, I can convert this PR to at least cross-reference (unless I missed those too)

Reply:
Yes, please do a cross reference.

Reply:
Done. Removed this entire section (seemed redundant, but did not thoroughly check).